rskj-core/src/test/java/co/rsk/peg/pegininstructions/PeginInstructionsBaseTest.java

```diff
@@ -0,0 +1,16 @@
+package co.rsk.peg.pegininstructions;
+
+import org.bouncycastle.util.encoders.Hex;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PeginInstructionsBaseTest {
+
+    @Test
+    public void extractProtocolVersion() {
+        int protocolVersion = PeginInstructionsBase.extractProtocolVersion(
+                Hex.decode("00010e537aad84447a2c2a7590d5f2665ef5cf9b667a"));
+
+        Assert.assertEquals(1, protocolVersion);
+    }
```

Comment:
Add a couple more tests, null and empty data at least.

Reply:
Add the case when a single byte is sent in the payload

Reply:
Maybe also a test when protocol version > 9 to check that it's parsing hex values to int correctly

Reply:
> Add the case when a single byte is sent in the payload

It is not possible that this method receive only one byte of data.

Reply:
what about:
>Add a couple more tests, null and empty data at least.

Reply:
extractOpReturnData will never return null or empty data.